src/components/application_manager/test/commands/mobile/simple_notification_commands_test.cc

```diff
@@ -68,6 +69,7 @@ typedef Types<commands::OnAppInterfaceUnregisteredNotification,
               commands::OnAudioPassThruNotification,
               commands::OnLanguageChangeNotification,
               commands::OnPermissionsChangeNotification,
+              commands::mobile::OnSeekMediaClockTimerNotification,
```

Comment:
@dchernievyes, you're right, but after review i'm going to squash my commits anyway